apps/chrome-devtools/src/app-devtools/localization-panel/localization-panel-pres.template.html

```diff
@@ -59,13 +59,15 @@ <h3 ngbAccordionHeader>
                         <input [attr.aria-describedby]="localization.key" class="form-control" type="text" [formControlName]="localization.key" />
                       </ng-container>
                       @if (localizationActiveStateOverridesForCurrentLang()[localization.key] || localizationLocalStateOverridesForCurrentLang()[localization.key]) {
+                        @let text = activeStateName() && localizationActiveStateOverridesForCurrentLang()[localization.key] !== localizationLocalStateOverridesForCurrentLang()[localization.key]
+                          ? 'Reset value to the one from ' + activeStateName() + ': ' + (localizationActiveStateOverridesForCurrentLang()[localization.key] || localization.value)
+                          : 'Reset value to the default one from metadata: ' + localization.value;
                         <i class="form-text text-warning fa-undo position-absolute end-0 me-3"
-                          [ngbTooltip]="
-                            activeStateName() && localizationActiveStateOverridesForCurrentLang()[localization.key] !== localizationLocalStateOverridesForCurrentLang()[localization.key]
-                              ? 'Reset value to the one from ' + activeStateName() + ': ' + (localizationActiveStateOverridesForCurrentLang()[localization.key] || localization.value)
-                              : 'Reset value to the default one from metadata: ' + localization.value
-                          "
-                          (click)="onLocalizationReset(localization)">
+                          [ngbTooltip]="text"
+                          [attr.aria-label]="text"
+                          tabindex="0"
+                          (click)="onLocalizationReset(localization)"
+                          (keyup.enter)="onLocalizationReset(localization)">
```

Comment:
same as mentioned by flo in https://github.com/AmadeusITGroup/otter/pull/2316/files#r1808617784